examples/customresourceinterpreter/webhook/app/workloadwebhook.go

```diff
@@ -26,7 +26,7 @@ type workloadInterpreter struct {
 
 // Handle implements interpreter.Handler interface.
 // It yields a response to an ExploreRequest.
-func (e *workloadInterpreter) Handle(ctx context.Context, req interpreter.Request) interpreter.Response {
+func (e *workloadInterpreter) Handle(_ context.Context, req interpreter.Request) interpreter.Response {
```

Comment:
How did you find these lint errors, what tools did you use?

Reply:
It's [GoLand Inspection](https://www.jetbrains.com/help/go/command-line-code-inspector.html#494cfa25). I don't know what tools it integrates, but it's very easy to find these lint errors.